### PR TITLE
LPS-154071 Notification Send

### PR DESCRIPTION
**Pull request by gustavolimav**
https://issues.liferay.com/browse/LPS-154071

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:sf

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryServiceImpl.java [hunk 1]
```diff
@@ -31,4 +33,22 @@
...
+	@Override
+	public void deleteNotificationQueueEntry(long notificationQueueEntryId)
+		throws PortalException {
+
+		notificationQueueEntryLocalService.deleteNotificationQueueEntry(
+			notificationQueueEntryId);
+	}
+
+	@Override
+	public NotificationQueueEntry resendNotificationQueueEntry(
```
Comment (gabrielwas) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryServiceImpl.java:46]: @gustavolimav can you remove method this from now? Nothing related to resend in this PR. Next one we do this.

File: modules/apps/notification/notification-rest-impl/rest-openapi.yaml [hunk 2]
```diff
@@ -63,6 +97,105 @@ info:
...
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/NotificationQueueEntry"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/NotificationQueueEntry"
+            tags: ["NotificationQueueEntry"]
+    "/notification-queue-entry/{notificationQueueEntryId}/resend":
```
Comment (gabrielwas) [inline modules/apps/notification/notification-rest-impl/rest-openapi.yaml:183]: @gustavolimav can you remove this one too?

**Review by gabrielwas — CHANGES_REQUESTED** [review]

File: modules/apps/notification/notification-service/service.xml [hunk 3]
```diff
@@ -40,6 +40,9 @@
 		<finder name="NotificationTemplateId" return-type="Collection">
 			<finder-column name="notificationTemplateId" />
 		</finder>
+		<finder name="Sent" return-type="Collection">
```
[the remark this answers is not in the document]
Comment (gustavolimav) [reply modules/apps/notification/notification-service/service.xml:43]: yeah, he is used when we send the email.

https://github.com/liferay-objects/liferay-portal/blob/4c1706f0d923bc60eb96c210b69c0ff705fef548/modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java#L129-L130

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java [hunk 4]
```diff
@@ -95,6 +108,98 @@ public NotificationQueueEntry deleteNotificationQueueEntry(
 		return notificationQueueEntry;
 	}
 
+	@Override
+	public NotificationQueueEntry resendNotificationQueueEntry(
```
[the remark this answers is not in the document]
Comment (gustavolimav) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java:119]: Sure

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryServiceImpl.java (hunk 1 above)
Comment (gustavolimav) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryServiceImpl.java:46]: Ok

File: modules/apps/notification/notification-rest-impl/rest-openapi.yaml (hunk 2 above)
Comment (gustavolimav) [reply modules/apps/notification/notification-rest-impl/rest-openapi.yaml:183]: Ok

**Pull request closed without merging**